source/style-guide/style/metadata.txt

```diff
@@ -0,0 +1,19 @@
+.. _metadata:
+
+=============
+Metadata
+=============
+
+.. default-domain:: mongodb
+
+.. include:: /includes/styles/corrections.rst
+
+Use ``.. meta::`` to add meta tags to a page. Adding meta tags aids in 
+searches both internally and via the Internet.
+
+.. code-block:: rst
```

Comment:
Can we put a subheader here that marks this as an Example? I can understand it just fine, but that's the pattern Kailie and I have been following.